cmd/dashboard/controller/common_page.go

```diff
@@ -275,13 +275,14 @@ func (cp *commonPage) terminal(c *gin.Context) {
 			}, true)
 			return
 		}
-
+		var cloudflareCookies string
+		cloudflareCookies, _ = c.Cookie("CF_Authorization")
```

Comment:
```suggestion
cloudflareCookies,  _ := c.Cookie("CF_Authorization")
```


建议 cloudflareCookies 如果不为空做一下基本的长度和格式的验证